### PR TITLE
Create README1.md

### PR DESCRIPTION
**Pull request by abedin-keshavarz234**
PU- GNN
This repository provides a framework of three stages that combines several drug features to predict DDS associated events, using attributed heterogeneous networks and multiple deep neural networks.  PU-GNN involves three main steps: 
1. Dimensionality reduction to extract optimal features: By reducing the feature dimensions, we aim to capture the most effective features for model evaluation.
 
 2. Addressing data uncertainty through positive unlabeled (PU) learning: We employ PU learning to identify positive and negative data from unlabeled data, reducing uncertainty and enhancing model accuracy. 
 
3. Predicting simultaneous side effects of drug pairs using a Graph Neural Network model: Our method predicts polypharmacy between two drugs efficiently using a neural graph network-based model.
 
 We implement this model which enables this model to be trained with GPUs. For additional details, read the published article for this work through this link.

Environments
Python 3.10 
numpy 1.21.4
pandas 1.1.3
sklearn 0.20

Usage
 You can access the data from GitHub link. When using this code, you need to clone this repo and load all the files in the folder into your running environment first. Then,Copy the PU_GNN_ run file and run it in your environment, all the files you want are in the project folder.
In the google colab environment, you can run the program by saving the project folder in the google drive environment.
Then copy and run the main.py file in the google colab environment

### CODE REVIEW

**Comment by Abedin-Keshavarz** [comment]
________________________________
From: Abedin-Keshavarz ***@***.***>
Sent: Sunday, May 19, 2024 4:05:30 AM
To: Abedin-Keshavarz/PU-GNN
Cc: Subscribed
Subject: [Abedin-Keshavarz/PU-GNN] Create README1.md (PR #6)


PU- GNN
This repository provides a framework of three stages that combines several drug features to predict DDS associated events, using attributed heterogeneous networks and multiple deep neural networks. PU-GNN involves three main steps:

  1.  Dimensionality reduction to extract optimal features: By reducing the feature dimensions, we aim to capture the most effective features for model evaluation.

  2.  Addressing data uncertainty through positive unlabeled (PU) learning: We employ PU learning to identify positive and negative data from unlabeled data, reducing uncertainty and enhancing model accuracy.

  3.  Predicting simultaneous side effects of drug pairs using a Graph Neural Network model: Our method predicts polypharmacy between two drugs efficiently using a neural graph network-based model.

We implement this model which enables this model to be trained with GPUs. For additional details, read the published article for this work through this link.

Environments
Python 3.10
numpy 1.21.4
pandas 1.1.3
sklearn 0.20

Usage
You can access the data from GitHub link. When using this code, you need to clone this repo and load all the files in the folder into your running environment first. Then,Copy the PU_GNN_ run file and run it in your environment, all the files you want are in the project folder.
In the google colab environment, you can run the program by saving the project folder in the google drive environment.
Then copy and run the main.py file in the google colab environment

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/Abedin-Keshavarz/PU-GNN/pull/6

Commit Summary

  *   42f2c29<https://github.com/Abedin-Keshavarz/PU-GNN/pull/6/commits/42f2c292563b476ba726dab9bfe89f9acff1541b> Create README1.md

File Changes

(1 file<https://github.com/Abedin-Keshavarz/PU-GNN/pull/6/files>)

  *   A README1.md<https://github.com/Abedin-Keshavarz/PU-GNN/pull/6/files#diff-40da93df987615bd9b854e3d6bd29cb7d656eb19839ef3296354177fc7416513> (20)

Patch Links:

  *   https://github.com/Abedin-Keshavarz/PU-GNN/pull/6.patch
  *   https://github.com/Abedin-Keshavarz/PU-GNN/pull/6.diff
